src/ui/layout/AboutDialog.vue

```diff
@@ -46,6 +52,9 @@ export default {
     methods: {
         showLicenses() {
             window.open('#/licenses');
+        },
+        openUserGuide() {
+            window.open('https://nasa.github.io/openmct/static/files/Open_MCT_Users_Guide.pdf');
```

Comment:
I wonder if this isn't considered covered because page2's codepath technically triggers this?

Reply:
Looks like this fails in ci because [Chromium does not support the internal PDF viewer in headless mode](https://github.com/microsoft/playwright/issues/6342#issuecomment-929183546), so the URL is never navigated to. The linked issue has a few workarounds for this.

Side note:
Should this test be moved to `branding.e2e.spec.js`? Or even just tacked on to the end of one of the existing branding tests that verify About modal contents?

Reply:
> Ah, if the e2e test were to pass, this would be covered! Let me take a look at why it's failing

@unlikelyzero Thank you :+1: It would be nice

Reply:
> The linked issue has a few workarounds for this
@ozyx Thanks. I will try workarounds :-)

> Should this test be moved to branding.e2e.spec.js?
Ok. I will do it

Reply:
maybe a better solution would be to use
<a href="path_to_pdf" target="_blank">

Reply:
@ozyx Hi,

I figured out the workaround (as described in the [issue](https://github.com/microsoft/playwright/issues/6342#issuecomment-929183546) ).
Unfortunately, the linters' checks were unsuccessful. Should I fix them?

Additionally, I moved the test to `branding.e2e.spec.js`.

What do you think about Pomaios' solutions?